### ISSUE TITLE
Problems with Example Files

### ISSUE THREAD

**Issue by daicollins** [jdf/Processing.py-Bugs#166]
A number of the example do not work. The principal problem is the failure to declare global variables as global within functions. I have remedied quite a number - is it permitted to submit?

**Comment by kazimuth** [comment]
Yes, thank you! Please make a PR at [the main repo](https://github.com/jdf/Processing.py).

**Comment by kazimuth** [comment]
You'll want to put the modified example files in the [mode/examples folder](https://github.com/jdf/processing.py/tree/master/mode/examples), unless you've been fixing standalone bugs, in which case they should go in the [examples.py](https://github.com/jdf/processing.py/tree/master/examples.py) folder.
If you haven't made a PR before, you should check out the PR guide [here](https://help.github.com/articles/using-pull-requests/).
